source-academy/frontend
# Achievement UI upgrade

### Description

- Staff are now able to see assessment achievements of the selected user
- The name at the top of the achievement dashboard is now the name of the selected user
- The cover image has been improved for better viewing
- Remove release and deadline of achievements now works

### Type of change


### How to test

Make sure you can create, delete, and edit achievements. Especially that changing the removing the deadline/release of an achievement gets properly saved in the backend.

Also make sure that changing selected user changes the achievements (and assessment achievements) shown in the dashboard.

### Checklist
